controller/src/beerocks/bml/bml_defs.h

```diff
@@ -587,6 +587,12 @@ typedef void (*BML_STATS_UPDATE_CB)(const struct BML_STATS_ITER *);
  */
 typedef void (*BML_EVENT_CB)(const struct BML_EVENT *);
 
+/**
+ * WFA_CA reply function. When registered, the function will be called after 
+ * the second reply of WFA-CA command. 
+ */
+typedef void (*BML_WFA_CA_CB)(const char *buffer);
```

Comment:
I think you should add length argument - We never pass a pointer to a buffer without a length in C code, according to SDL.

Reply:
> I think we could have used an event instead, but adding a new callback type is OK as well.
BML events are actually the same. They use callback function as well. The use of them is for Beerocks notifications when something happened on the system and has one register function for all events. It is not fit here.

Reply:
> I think you should add length argument - We never pass a pointer to a buffer without a length in C code, according to SDL.

It is not needed. I pass to the callback const char* so it cannot be modified.
The size is also irrelevant because the string just being printed.
No security holes here. Guarantee!